ChangeLog

```diff
@@ -1,6 +1,10 @@
 CHANGES
 =======
 
+v1.1.8
+-------
+Upgrade plugin for compatibility with Mitaka version (including docker) [#43]
```

Comment:
Should Netwon be mentioned also in this line?

Reply:
updated